src/FcsWatch.Binary/AutoReload.fs

```diff
@@ -17,7 +17,7 @@ open System.Text
 
 [<RequireQualifiedAccess>]
 type WhyRun =
-    | Rerun
+    | Rerun of fileChanges: string list
```

Comment:
`fileChanges` should be renamed to dllUpdates?

### As in situation 
https://github.com/humhei/FCSWatch/pull/21/files#diff-51eead0cd0a70087243000224967562aR559
a filechange means a sourceFileChanges(file end with .fs)

### In current situation
a filechange means a dllUpdate(file end with .dll)

Reply:
Fixed/updated